### DISCUSSION TITLE
"forbidden" message.

### DISCUSSION THREAD

**Discussion by Dhexio** [ppy/osu#22687] [Q&A] [answered]
Can anyone help me? I'm trying to log in to my osu!lazer account but it's showing a **"forbidden"** message.

**Comment by mk56-spn** [comment]
I think there was an issue with restricted accounts not being able to log in, could be that if you're account isnt "clean".

**Reply by Dhexio** [reply]
I have never done anything prohibited. Why is my account not "clean" even though I only downloaded beatmaps through the application?"

**Comment by bdach** [comment]
Please attach an actual screenshot of the actual error you are seeing. Actual logs would also be appreciated (https://github.com/ppy/osu/discussions/15188).

**Reply by Dhexio** [reply]
[performance.log](https://github.com/ppy/osu/files/10778894/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10778895/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10778896/updater.log)
[database.log](https://github.com/ppy/osu/files/10778897/database.log)
[input.log](https://github.com/ppy/osu/files/10778898/input.log)
[network.log](https://github.com/ppy/osu/files/10778899/network.log)

**Comment by cdwcgt** [comment, marked as answer]
This looks like a human-machine verification of cloudflare, maybe you are using a public network or other problems, changing the network may solve your problem.
Or try CAPTCHA on lazer.ppy.sh? Probably won't solve the problem

I guess it is due to the opening of the new endpoint (lazer.ppy.sh) that the api node needs human-machine verification.

**Reply by Dhexio** [reply]
I was able to log in to the website OSU, but it's forbidden in the application

**Reply by cdwcgt** [reply]
how about https://lazer.ppy.sh
also https://lazer.ppy.sh/oauth/token
Normally, you should see this
![image](https://user-images.githubusercontent.com/34775378/220328457-ae5f5860-4834-4259-be6d-135c4e0f891e.png)
not this
![image](https://user-images.githubusercontent.com/34775378/220328661-972b2ad0-7309-4636-bddf-ed222fe478b9.png)

**Reply by Dhexio** [reply]
![image](https://user-images.githubusercontent.com/125784035/220520034-c365b19a-24a6-4a9c-b55b-dd221a81c635.png)
![image](https://user-images.githubusercontent.com/125784035/220520135-53550692-d682-4ca5-8cd5-d3ed399ebb60.png)

application
![image](https://user-images.githubusercontent.com/125784035/220520151-0e6eb1fa-aa05-41c1-9a0b-2591df10b919.png)

**Reply by cdwcgt** [reply]
You marked it as resolved, is it ok? If there is still a problem, I want to check the log.

**Reply by Dhexio** [reply]
I don't know what happened, all I did was wait and then I was able to log back into the application. Haha thank you for your help
